config/packit/orderly-web.yml

```diff
@@ -141,8 +141,8 @@ packit:
     name: packit-api
     tag: main
   app:
-    name: packit
-    tag: mrc-4261
+    name: montagu-packit
+    tag: mrc-5140-trusted-headers
```

Comment:
how do you want to handle the merge order here? Keep this for now as https://github.com/mrc-ide/packit/pull/187 is in draft?

Reply:
Yeah, for the moment. Looks like that's what we did before!